quasar-core/src/main/java/co/paralleluniverse/fibers/Fiber.java

```diff
@@ -2202,7 +2209,7 @@ public void write(Kryo kryo, Output output, Fiber<?> f) {
                 try {
                     f.stack.resumeStack();
                     kryo.writeClass(output, f.getClass());
-                    new FieldSerializer<>(kryo, f.getClass()).write(kryo, output, f);
+                    getFieldSerializer(kryo, f.getClass()).write(kryo, output, f);
```

Comment:
Oh FFS! So getting rid of "raw" types also requires:
```java
(Class<? extends Fiber<?>>)f.getClass()
```
and adding "unchecked" to this function's `@SuppressWarnings`.